.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+---
+# yamllint disable rule:line-length
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - main
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create Tag
+        uses: actions/github-script@v5
```

Comment:
Would it make sense to switch this to new actions-collection/create-release-tags action?